lib/const.rb

```diff
@@ -70,20 +69,25 @@
 CREW_SHRINK_ARCHIVE = ENV['CREW_SHRINK_ARCHIVE']
 
 # Set testing constants from environment variables
-crew_testing_repo = ENV['CREW_TESTING_REPO']
-crew_testing_branch = ENV['CREW_TESTING_BRANCH']
-crew_testing = ENV['CREW_TESTING']
-crew_testing = '0' if crew_testing_repo.nil? || crew_testing_repo.empty?
-crew_testing = '0' if crew_testing_branch.nil? || crew_testing_branch.empty?
-CREW_TESTING = crew_testing
-CREW_TESTING_BRANCH = crew_testing_branch
-CREW_TESTING_REPO = crew_testing_repo
+CREW_TESTING_BRANCH = ENV['CREW_TESTING_BRANCH']
+CREW_TESTING_REPO = ENV['CREW_TESTING_REPO']
+
+CREW_TESTING = unless CREW_TESTING_BRANCH.to_s.empty? or CREW_TESTING_REPO.to_s.empty?
+  ENV['CREW_TESTING']
+else
+  '0'
+end
```

Comment:
Lines 75-79 could be turned into an immediate if one-liner.

Reply:
Done